test/deploy/kubernetes/manual/deploy.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+SCRIPT_DIR=$(dirname $0)
+MAXNODES="3"
+NAMESPACE="cortx"
+
+function print_header {
+    echo -e "---------------------------------------------------"
+    echo -e "$1"
+    echo -e "---------------------------------------------------"
+}
+
+# Create NameSpace
+kubectl create namespace $NAMESPACE
+
+# Create ConfigMap
+kubectl create configmap solution-config \
+    --from-file=$SCRIPT_DIR/solution-config/cluster.yaml \
+    --from-file=$SCRIPT_DIR/solution-config/config.yaml \
+    --namespace $NAMESPACE
+
+#Create Storage Volumes
+kubectl apply -f $SCRIPT_DIR/persistent-volumes/cortx-config-pv.yaml  --namespace $NAMESPACE
```

Comment:
We have to create storage volumes for all nodes.... This needs to be removed?

Reply:
This are Local directory Volumes Which will needs be mounted on all nodes right? (etc-cortx, var-data-cortx, var-log-cortx)